package.json

```diff
@@ -76,83 +76,83 @@
     "type": "git",
     "url": "git+https://github.com/BigRoomStudios/strangeluv.git"
   },
-  "author": "David Zukowski <david@zuko.me> (http://zuko.me)",
   "license": "MIT",
   "dependencies": {
-    "axios": "^0.12.0",
-    "babel-cli": "^6.5.1",
-    "babel-core": "^6.3.17",
-    "babel-loader": "^6.2.0",
-    "babel-plugin-transform-runtime": "^6.3.13",
-    "babel-polyfill": "^6.9.0",
-    "babel-preset-es2015": "^6.3.13",
-    "babel-preset-react": "^6.3.13",
+    "axios": "^0.18.0",
```

Comment:
Dunno how you feel about managing deps or not, but I think I started liking the `x`'s and removing the `^` carets
"axios": "^0.18.0"   —>  "axios": "0.18.x"